tests/components/glances/test_config_flow.py

```diff
@@ -1,38 +1,26 @@
 """Tests for Glances config flow."""
-from unittest.mock import patch
+from unittest.mock import MagicMock
 
-from glances_api import exceptions
+from glances_api.exceptions import GlancesApiConnectionError
 import pytest
 
-from homeassistant import config_entries, data_entry_flow
+from homeassistant import config_entries
 from homeassistant.components import glances
-from homeassistant.const import CONF_SCAN_INTERVAL
 from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import (
+    RESULT_TYPE_ABORT,
```

Comment:
Please use the enum instead. The other constants are deprecated.

https://github.com/home-assistant/core/blob/1d9bbdcbf3d1d677f50d2c1b6a0f7568a2a20281/homeassistant/data_entry_flow.py#L23